x-pack/test/functional/services/ml/job_wizard_common.ts

```diff
@@ -628,5 +628,69 @@ export function MachineLearningJobWizardCommonProvider(
         await testSubjects.existOrFail(expectedSelector);
       });
     },
+
+    async assertAnnotationRecommendationCalloutVisible(expectVisible: boolean = true) {
+      const callOutTestSubj = 'mlJobWizardAlsoEnableAnnotationsRecommendationCallout';
+      if (expectVisible)
+        await testSubjects.existOrFail(callOutTestSubj, {
+          timeout: 3_000,
+        });
+      else
+        await testSubjects.missingOrFail(callOutTestSubj, {
+          timeout: 3_000,
+        });
+    },
+
+    async goToTimeRangeStep() {
+      await retry.tryForTime(60_000, async () => {
+        await testSubjects.existOrFail('mlJobWizardTimeRangeStep');
+        await testSubjects.click('mlJobWizardTimeRangeStep');
+        await testSubjects.existOrFail('mlJobWizardStepTitleTimeRange');
+      });
+    },
+
+    async setShortDurationTimeRange(shortDurationEndDate: string) {
+      // set the new end date
+      await testSubjects.setValue('mlJobWizardDatePickerRangeEndDate', shortDurationEndDate, {
+        clearWithKeyboard: true,
+        typeCharByChar: true,
+      });
+
+      // click away from time popover
+      await this.goToTimeRangeStep();
```

Comment:
if you want to escape popover, it would be a better option 
```suggestion
  // escape popover
  await browser.pressKeys(browser.keys.ESCAPE);
```